pkg/controllers/provisioning/scheduling/topologygroup.go

```diff
@@ -83,6 +84,12 @@ func NewTopologyGroup(topologyType TopologyType, topologyKey string, pod *v1.Pod
 		domains:    domainCounts,
 		owners:     map[types.UID]struct{}{},
 	}
+
+	if minDomains != nil {
+		topologyGroup.minDomains = minDomains
```

Comment:
optional: why not just assign the value directly?

Reply:
I don't know why not!